ws22/fpga-os/coyote.nix

```diff
@@ -1,37 +1,69 @@
-with import <nixpkgs> {}; 
+{ pkgs ? import (fetchTarball "https://github.com/NixOS/nixpkgs/archive/1e383aada51b416c6c27d4884d2e258df201bc11.tar.gz") {}
```

Comment:
I pinned now the commit for reproducibility. This definitely works on ryan:

```
mkdir hw/build && cd hw/build
cmake .. -DFDEV_NAME=u50 -DEXAMPLE=perf_fpga
make shell compile
```

Please reproduce the error on this machine

jinja2 can also imported in this shell without PYTHONPATH set:

```
[joerg@ryan fpga-os]$ PYTHONPATH= python -mjinja2
/nix/store/zlj86pbv78y3vmwv4lra2n0dbssg8sh5-python3-3.10.10-env/bin/python3.10: No module named jinja2.__main__; 'jinja2' is a package and cannot be directly executed
```

Reply:
also make sure you actually have this branch checkout.

Reply:
There's still the jinja2 error on ryan. I ran `PYTHONPATH= python -mjinja2` but it didn't solve the error?

Reply:
To reproduce the other error on ryan.

```
cd Coyote # make sure you are on the latest branch
mkdir build_perf_rdma_host_hw && cd build_perf_rdma_host_hw
cmake ../hw/ -DFDEV_NAME=u280 -DEXAMPLE=perf_rdma_host -DACLK_F=250
make shell

```

Reply:
This is not the command you gave me.

Reply:
> There's still the jinja2 error on ryan. I ran `PYTHONPATH= python -mjinja2` but it didn't solve the error?

This was too prove that python can import jinja2.

Reply:
Hi both, 

This is just to let you know that I tried building the code with the new updates on both ryan and amy. 
I am able to reproduce the exact same error that @jedichen121 posted before. The problem looks like to be with `-DEXAMPLE=perf_rdma_host`. The default example (perf_fpga) compiles successfully. However, we would like to have the rdma example up and running too.

Thanks,
Dimitra

Reply:
Hi is there any update on this?